core/misc/dialog.js

```diff
@@ -75,7 +75,20 @@ Backdrop.dialog = function (element, options) {
         })
         .dialog('widget').css('position', 'fixed');
       Backdrop.optimizedResize.add(resetPosition, 'dialogResize.' + dialogId);
-      resetPosition();
+
+      // Because of Chromium's behavior, we must wait until the ajax-loaded
+      // jquery.ui.dialog.css is applied.
+      var waitForCss = function() {
+        var cssOverflowProperty = getComputedStyle($element[0]).getPropertyValue('overflow');
```

Comment:
In my original PR I had this variable assignment outside the function. Scope-wise, this approach may be better, as it keeps the variable within the scope of the closure.  I approached this differently because one of the features of `getComputedStyle()` is that it creates a live object "which updates automatically when the element's styles are changed" according to the [docs](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle). So, this way, we avoid making a new assignment every time the function is invoked. 

Both approaches work, and I guess it's just a matter of preference.

Reply:
Good idea. I didn't know that about `getComputedStyle()`. I moved it to a separate variable again. :+1: